report/completeness.html

```diff
@@ -89,7 +89,7 @@ <h1 id="countryHeader">${selectCountry}</h2>
 
     async function getStatus(_country, _countryValue) {
         fetchError = "No data for selected country!"
-        await fetch("completeness_" + _countryValue + ".json").then((response) => {
+        await fetch("../completeness_report/completeness_" + _countryValue + ".json").then((response) => {
```

Comment:
where do You want this dir to be? `../` in dir is very confusing
better `report/completeness_report/completeness_...`

Reply:
agreed on using completeness_report folder.
js can reach by ../completeness_report